Topics/Tech_Stacks/Iframe.md

```diff
@@ -0,0 +1,25 @@
+# Learning Iframe
+
+## Introduction
+
+An Iframe is a html element that can let programmer to display a child website inside a parent website. It is supported by many browsers, like chrome, firefox and safari.
+
+## Why use Iframe
+Sometimes we want to let user to interact with a webpage without open a new webpage tage and redict to it. For example, we want to embed a survey inside a webpage so that the user can conveniently fill out the survey after reading the previous content in this webpage.
+
+## How to use Iframe
+You can add a html element inside your html file following this format <iframe src="some like" title="some title"></iframe>
```

Comment:
```suggestion
You can add an iframe element inside your HTML file by following this format:
`<iframe src="some like" title="some title"></iframe>`
```

Wrap code snippets with `` to get code formatting. I would also suggest updating the src attribute to an actual URL so it helps the reader understand what it is — "some like" isn't very helpful. Also, please check for typos and grammatical errors.